### ISSUE TITLE
some missings types

### ISSUE THREAD

**Issue by 3C0D** [Fevol/obsidian-typings#91]
import 'obsidian'

declare module "obsidian" {
    interface WorkspaceSplit {
        containerEl: HTMLElement
    }
    interface WorkspaceSidedock {
        containerEl: HTMLElement
        setSize(width: number): void
    }

    interface View {
        file: TFile,
    }
}

**Comment by mnaoumov** [comment]
Implemented in https://github.com/Fevol/obsidian-typings/releases/tag/2.2.1-beta.30

**Comment by 3C0D** [comment]
@mnaoumov 
For Your Information I've created a template that significantly accelerates the process of creating, modifying, and publishing Obsidian plugins on GitHub. This template incorporates obsidian-typings to enhance development capabilities.
After numerous iterations, the template has reached a highly refined state. However, I believe there's still considerable room for improvement. I created a new branch for Sass but I need to update it. I welcome any feedback, suggestions, or contributions to further enhance this template. I think it will be a good thing to do a repo with several possible base configurations.
https://github.com/3C0D/obsidian-sample-plugin-modif
